images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1

```diff
@@ -25,6 +25,11 @@ function Get-SQLCmdVersion {
     return "sqlcmd $sqlcmdVersion"
 }
 
+function Get-SqlPackageVersion {
+    $sqlPackageVersion = sqlpackage /version | Take-OutputPart -Part 0
```

Comment:
nitpick: Looks like we don't need `Take-OutputPart `at all
![image](https://user-images.githubusercontent.com/48208649/125041751-c6869f80-e0a1-11eb-89a5-1f94ef91c134.png)
```suggestion
    $sqlPackageVersion = sqlpackage /version
```

Reply:
done